### PR TITLE
Rename processor

### PR DESCRIPTION
**Pull request by meganelacheny**
- Update attributes table of processor object
- Add example of Expression processor with label

### CODE REVIEW

**Review by fpassaniti — APPROVED** [review]
👍 👌

File: source/includes/_processing.md [hunk 1]
```diff
@@ -22,11 +22,26 @@ The processor object is composed of a unique ID for the processor, the processor
 }
 ```
 
+> Example object with label
+
+```json
+{
+    "processor_uid": "pr_mgnlcn",
+    "name":"expression",
```
Comment (NTerpo) [inline source/includes/_processing.md]:
```suggestion
    "name" : "expression",
```

File: source/includes/_processing.md [hunk 2]
```diff
@@ -22,11 +22,26 @@ The processor object is composed of a unique ID for the processor, the processor
...
+    "label":"my_column > or < 0",
```
Comment (NTerpo) [inline source/includes/_processing.md]:
```suggestion
    "label" : "my_column > or < 0",
```

File: source/includes/_processing.md [hunk 3]
```diff
@@ -22,11 +22,26 @@ The processor object is composed of a unique ID for the processor, the processor
 }
 ```
 
+> Example object with label
+
+```json
+{
+    "processor_uid": "pr_mgnlcn",
```
Comment (NTerpo) [inline source/includes/_processing.md]:
```suggestion
    "processor_uid" : "pr_mgnlcn",
```

File: source/includes/_processing.md [hunk 4]
```diff
@@ -22,11 +22,26 @@ The processor object is composed of a unique ID for the processor, the processor
...
+    "name":"expression",
+    "label":"my_column > or < 0",
+    "args": {
```
Comment (NTerpo) [inline source/includes/_processing.md]:
```suggestion
    "args" : {
```

File: source/includes/_processing.md [hunk 5]
```diff
@@ -22,11 +22,26 @@ The processor object is composed of a unique ID for the processor, the processor
...
+        "output_field": "result",
```
Comment (NTerpo) [inline source/includes/_processing.md]:
```suggestion
        "output_field" : "result",
```

File: source/includes/_processing.md [hunk 6]
```diff
@@ -22,11 +22,26 @@ The processor object is composed of a unique ID for the processor, the processor
...
+        "expression": "{{my_column > 0 ? 'positive' : 'negative'}}"
```
Comment (NTerpo) [inline source/includes/_processing.md]:
```suggestion
        "expression" : "{{my_column > 0 ? 'positive' : 'negative'}}"
```

**Review by NTerpo — APPROVED** [review]
I know I've been a bit pedantic but it just make the json more readable

**Pull request merged**